components/style/values/specified/box.rs

```diff
@@ -352,12 +393,14 @@ impl Display {
         match self.outside() {
             DisplayOutside::Inline => {
                 let inside = match self.inside() {
+                    #[cfg(any(feature = "servo-layout-2013", feature = "gecko"))]
```

Comment:
`#[cfg]`ing this out is wrong, why?

Reply:
I can't keep it, I removed `DisplayInside::Block` and `DisplayInside::Inline`. Those are hacks to make layout supposedly easier, but it's just hindering layout 2020.

Reply:
I added back a blockification which makes it store `DisplayInside::FlowRoot` with `servo-layout-2020`.

Reply:
Err, I meant I made it store `DisplayInline::Flow`, sorry.